packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -508,11 +567,23 @@ export async function launch({
             `Cannot run a bootstrap block at height ${computedHeight}`,
           );
         }
+        const blockHeight = 0;
+        const runNum = 0;
         controller.writeSlogObject({
           type: 'cosmic-swingset-bootstrap-block-start',
           blockTime,
         });
-        await processAction(action.type, bootstrapBlock);
+        controller.writeSlogObject({
+          type: 'cosmic-swingset-run-start',
+          blockHeight,
+          runNum,
+        });
+        await bootstrapBlock();
```

Comment:
The last commit ("Depth first scheduling") changes this line from `await processAction(action.type, bootstrapBlock)` to `await bootstrapBlock()`. Should that change have happened in an earlier commit? It seems to me that this specific commit should only be adding the `cosmic-swingset-run-start` logs around a call that should have been the same as in the previous commit. (The end result might be the same, and seems correct, but the path by which the commits get there might be slightly off).

Reply:
Good catch, it should not have happened at all. I'm using `processAction` to `console.log` any error. The `runTime` measurement in this case are simply dropped.